### ISSUE TITLE
Project Inventory: BallotNav

### ISSUE THREAD

**Issue by Olivia-Chiong** [hackforla/product-management#206]
### Overview
We need to make an inventory for this project so we understand what assistance this project needs and what examples of effective processes or practices this team might have.

### Action Items
- [ ] Check the project for the items on the check list below
- [ ] Mark off the items when you find it and include the links where available
- [ ] Provide comments on what you find in the issue and also highlight when they have a great example so we can use it for the organization

**Product**
- [ ] [Readme]()
- [ ] [Wiki]()
- [ ] [Google Drive]()
- [ ] [GitHub Repository]()
   - [ ] [Project Board]()
   - [ ] [Issue Templates]() 
- [ ] [Slack Channel]()
- [ ] 1Password
- [ ] Project Email
   - [ ] projectname@hackforla.org
   - [ ] projectname@gmail.com (You will need this for setting up some applications like MIro that use rate limiting. Do not send emails using this address) 
- [ ] [Team Roster]()
- [ ] [Product One Sheet]()
- [ ] [Recruitment Form]()
- [ ] [OKRs and WBS]()
- [ ] [Roadmap]()
- [ ] [Storypoints]()
- [ ] [Meeting Minutes Template]()

**Marketing**
   - [ ] [Marketing Objectives]()
      - [ ] Create new issue using this template: [Define Marketing Objectives for [INSERT PROJECT NAME]](https://github.com/hackforla/marketing/issues/new?assignees=&labels=Level%3A+Learning%2C+Skill%3A+Brand+Strategy%2C+Skill%3A+Digital+Marketing%2C+Skill%3A+Media+Strategy%2C+documentation&template=define-marketing-objectives-for--insert-project-name-.md&title=Define+Marketing+Objectives+for+%5BINSERT+PROJECT+NAME%5D+project)
      - [ ] Copy the link for the issue into the parentheses above
   - [ ] [Collect Marketing Assets]()
      - [ ] Create new issue using this template: [Collect Marketing Assets for [INSERT PROJECT NAME]](https://github.com/hackforla/marketing/issues/new?assignees=&labels=feature%3A+marketing+inventory+checklist&template=collect-marketing-assets-for--insert-project-name-.md&title=Collect+Marketing+Assets+for+%5BINSERT+PROJECT+NAME%5D)
      - [ ] Copy the link for the issue into the parentheses above
   
**UI/UX**
- [ ] [Figma]()
- [ ] [Miro]()
- [ ] [Presentation Deck]()
- [ ] [Competitive Analysis]()
- [ ] [User Persona/Journeys]()
- [ ] [Communication Style Guide (including documents, spreadsheets and forms)]()
- [ ] [User Research Guide/Template]()
- [ ] [Wireframes Template]()
- [ ] [Website analytics software setup]()

### Resources/Instructions
REPLACE THIS TEXT -If there is a website which has documentation that helps with this issue provide the link(s) here.

**Comment by Olivia-Chiong** [comment]
Refer to https://github.com/hackforla/product-management/issues/90#issuecomment-774725376 for current project leadership

**Comment by Rabia77** [comment]
I did find the links to the Team Roster and Google Drive but I was denied access. I have pasted the links anyway. Please advise.

**Comment by Rabia77** [comment]
On the github repository:
5. Add yourself to the Team Roster. Then, Slack our PM with "Added to Roster".
7. Read project details and review our Project Management Board
The links to both Team Roster and Project Management Board lead to the Volunteer Signup Page. 
Needs to be fixed?

**Comment by Olivia-Chiong** [comment]
For one sheet, please use PDF @Rabia77

**Comment by kcoronel** [comment]
@Rabia77 thanks for the tag. I am typing below some responses/comments from your post above. 
1. You can give us your email address and we can get you access to our drive 
2. Repository and read me are updated, thank you! 
3. We will take a look at other projects wiki and attempt to recreate something similar

**Comment by Olivia-Chiong** [comment]
@Rabia2219 This project has other assets such as competitive analysis template etc. Could you please do a more in depth check?

**Comment by Rabia2219** [comment]
> @Rabia2219 This project has other assets such as competitive analysis template etc. Could you please do a more in depth check?

Yes...will do.

**Comment by Rabia2219** [comment]
Hello @kcoronel,
I wanted to ask if the BallotNavPresentation.gslides in the BallatNav drive refers to the Presentation Deck? If yes, it is incomplete so do you still want me to add the link up here in the Project Inventory issue?
Thanks

**Comment by kcoronel** [comment]
@Rabia2219 yes, they are incomplete so I would rather not add it to this inventory sheet. Thanks for checking in.
